certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1557 @@
+          "format": "double",
+          "description": "Gets or Sets EarnestMoney",
+          "type": "number"
+        },
+        "downPayment": {
+          "format": "double",
+          "description": "Gets or Sets DownPayment",
+          "type": "number"
+        },
+        "ownerPolicyLiabilityOverride": {
+          "format": "double",
+          "description": "Gets or Sets OwnerPolicyLiabilityOverride",
+          "type": "number"
+        },
+        "loanPolicyLiabilityOverride": {
+          "format": "double",
+          "description": "Gets or Sets LoanPolicyLiabilityOverride",
+          "type": "number"
+        },
+        "cashOutAmount": {
+          "format": "double",
+          "description": "Gets or Sets CashOutAmount",
+          "type": "number"
+        },
+        "loans": {
+          "description": "Gets or Sets Loans",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.Loan1"
+          }
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.PropertyType": {
+      "format": "int32",
+      "description": "Gets or Sets PropertyType",
+      "enum": [
+        0,
+        1,
+        2,
+        3,
+        4,
+        5,
+        6,
+        7,
+        8,
+        9,
+        10
+      ],
+      "type": "integer"
+    },
+    "PubCS.TitleOrder.Api.Models.AddressType": {
+      "type": "string"
+    },
+    "PubCS.TitleOrder.Api.Models.Address": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "type": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.AddressType"
+        },
+        "line1": {
+          "description": "Gets or Sets Line1",
+          "type": "string"
+        },
+        "line2": {
+          "description": "Gets or Sets Line2",
+          "type": "string"
+        },
+        "city": {
+          "description": "Gets or Sets City",
+          "type": "string"
+        },
+        "state": {
+          "description": "Gets or Sets State",
+          "type": "string"
+        },
+        "zip": {
+          "description": "Gets or Sets Zip",
+          "type": "string"
+        },
+        "county": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.County"
+        },
+        "country": {
+          "description": "Gets or Sets Country",
+          "type": "string"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.Tax": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "category": {
+          "description": "Gets or Sets Category",
+          "type": "string"
+        },
+        "payee": {
+          "description": "Gets or Sets Payee",
+          "type": "string"
+        },
+        "amount": {
+          "format": "double",
+          "description": "Gets or Sets Amount",
+          "type": "number"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.Property": {
+      "description": "",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.